probe-rs/targets/MSPM0C_Series.yaml

```diff
@@ -0,0 +1,146 @@
+name: MSPM0C Series
+generated_from_pack: true
+pack_file_release: 1.1.1
+variants:
+- name: MSPM0C1103
+  cores:
+  - name: main
+    type: armv6m
+    core_access_options: !Arm
+      ap: 0
+      psel: 0x0
+  memory_map:
+  - !Nvm
+    name: IROM1
+    range:
+      start: 0x0
+      end: 0x2000
+    cores:
+    - main
+    access:
+      write: false
+      boot: true
+  - !Ram
+    name: IRAM1
+    range:
+      start: 0x20000000
+      end: 0x20000400
+    cores:
+    - main
+  - !Generic
+    name: NonMain
+    range:
+      start: 0x41c00000
+      end: 0x41c00200
+    cores:
+    - main
+    access:
+      write: false
+      execute: false
+  - !Generic
+    name: Factory
+    range:
+      start: 0x41c40000
+      end: 0x41c40080
+    cores:
+    - main
+    access:
+      write: false
+      execute: false
+  flash_algorithms:
+  - mspm0c110x_main_16kb
+  - mspm0c110x_nonmain
+- name: MSPM0C1104
+  cores:
+  - name: main
+    type: armv6m
+    core_access_options: !Arm
+      ap: 0
+      psel: 0x0
+  memory_map:
+  - !Nvm
+    name: IROM1
+    range:
+      start: 0x0
+      end: 0x4000
+    cores:
+    - main
+    access:
+      write: false
+      boot: true
+  - !Ram
+    name: IRAM1
+    range:
+      start: 0x20000000
+      end: 0x20000400
+    cores:
+    - main
+  - !Generic
+    name: NonMain
+    range:
+      start: 0x41c00000
+      end: 0x41c00200
+    cores:
+    - main
+    access:
+      write: false
+      execute: false
+  - !Generic
+    name: Factory
+    range:
+      start: 0x41c40000
+      end: 0x41c40080
+    cores:
+    - main
+    access:
+      write: false
+      execute: false
+  flash_algorithms:
+  - mspm0c110x_main_16kb
+  - mspm0c110x_nonmain
+flash_algorithms:
+- name: mspm0c110x_main_16kb
+  description: MSPM0C110X MAIN 16KB
+  default: true
+  instructions: gLUJSAFoByKRQwFgB0gBaA8ikUMBYADw5fgFSAFoBCKRQwFgACCAvQATQEAEAQtAEO0A4AAgcEf4tQRGEUgAaBFJAUCgChFKAkCKQhfSASBCAg9OMEYhRgDwpvgEIMdDACUoRsAHC9EALwnQQCIwRiFGAPBf+H8cAPAM+PLnASUoRvi9GADEQf8PAAD/7z8AANAMQARIAWjJB/zQAGgCIQFASAhwR8BG0OMMQIC1A0gA8Ez4ASFIQIC9wEYA0AxA+LUURg1GBkYVSABoFUkBQLAKFUoCQAEgikIg0gAtHtAHISlAG9EBIRFPACAALRbQyQcU0AEgQgIMSDFGAPBU+DhoQAf81AlIMUYiRgDwOvgINgg0CD3/973/AUbl5/i9GADEQf8PAAD/7z8AANAMQNDjDEAQtQIjE0MRIhIChBhjYCFiASGBUBC9wEYQtQtJACIKYNNDC2QRIQkCQxhSJFxgGmIBIkJQBUlAGAFoyQf80ABoAiEBQEgIEL3Q4QxA0BMAAHC1ESMbAsQYASVlYAROZmIhYhFoIWNRaGFjxVBwvcBG/wEAABC1DUvAGAEjnAKiQgzQXAKiQhDRiQIJDR8pCNn/KQrYAB0gOckIAuBAMEkEyQ6LQAFomUMBYBC90BEAAAEggQIGShNoi0MTYAAFUWiBQ1FgA0gBaIkD/NRwR8BGAAELQAQCC0AAAAAA
+  load_address: 0x20000020
+  pc_init: 0x1
+  pc_uninit: 0x35
+  pc_program_page: 0xc1
+  pc_erase_sector: 0x39
+  pc_erase_all: 0xad
+  data_section_offset: 0x20c
+  flash_properties:
+    address_range:
+      start: 0x0
+      end: 0x4000
+    page_size: 0x40
+    erased_byte_value: 0xff
+    program_page_timeout: 500
+    erase_sector_timeout: 3000
+    sectors:
+    - size: 0x400
+      address: 0x0
+  stack_size: 256 # Manually added
```

Comment:
Comments will not be preserved by the tools, add this to the description instead, see my other comment.